### PR TITLE
improving the driver re: the checking of input pathnames, improving error messages, trying to prevent misleading error messages.

### PR DESCRIPTION
**Pull request by Abe149**

### CODE REVIEW

**Comment by mbudiu-vmw** [comment]
Haven't I reviewed a version of this code?

File: tools/driver/p4c_src/setup.cfg___for_pycodestyle [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[pycodestyle]
```
Comment (fruffy) [inline tools/driver/p4c_src/setup.cfg___for_pycodestyle:1]: Can you explain the purpose of this file? Is this a style file only for the driver code?

File: tools/driver/p4c_src/setup.cfg___for_pycodestyle [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+###   E262: inline comment should start with '# '
+###   E266: too many leading '#' for block comment
+###   E303: too many blank lines (3)
+###
+### Abe disabled E262 and E266 because he thinks it`s nice to use “###” for a real comment, leaving plain “# ” for commented-out code.
+###
+### Abe disabled E303 because he likes to use blank lines to visually separate conceptually-separate code.
+###   Maybe he should use comment lines instead, e.g. “### ——————————————————————————————————————————— ###”?
+
+# max-line-length = 160
```
Comment (fruffy) [inline tools/driver/p4c_src/setup.cfg___for_pycodestyle:18]: 160 might be excessive. 80-100 characters is a convenient width for window tiling and for laptops.

File: tools/driver/p4c_src/setup.cfg___for_pycodestyle (hunk 1 above)
Comment (Abe149) [reply tools/driver/p4c_src/setup.cfg___for_pycodestyle:1]: Yup.

File: tools/driver/p4c_src/setup.cfg___for_pycodestyle (hunk 2 above)
Comment (Abe149) [reply tools/driver/p4c_src/setup.cfg___for_pycodestyle:18]: The 160 is commented out.

**Comment by Abe149** [comment]
@mbudiu-vmw 

> Haven't I reviewed a version of this code?

Yes.  I thought it best to open a new review, for [a] reason[s] I can explain to you via email if you like.

**Comment by mbudiu-vmw** [comment]
Review takes a lot of time. It is not productive to review the same code again.
If you make changes they should be separate commits which can be reviewed separately.

**Review by hanw — APPROVED** [review]

**Pull request merged**